### PR TITLE
Add integration test via localstack

### PR DESCRIPTION
**Pull request by jderusse**
This adds localstack and enable integration tests for:
* sts
* cloudformation
* cloudwatch logs
* events bridge
* iam
* sns
* ssm

When possible, I migrate existing services to localstack:
* dynamodb 

Other services can't be migrated because use a specific set of feature that are not provided by localstack
* s3
* sqs
* lambda

### CODE REVIEW

File: src/Core/composer.json [hunk 1]
```diff
@@ -18,7 +18,7 @@
         "psr/cache": "^1.0",
         "psr/log": "^1.0",
         "symfony/deprecation-contracts": "^2.1",
-        "symfony/http-client": "^4.4 || ^5.0,!=5.2.0",
+        "symfony/http-client": "^4.4.16 || ^5.1.7,!=5.2.0",
```
Comment (jderusse) [inline src/Core/composer.json:21]: Not really related to this PR, but the PR reveal the bug.
I old versions (prior this PR https://github.com/symfony/symfony/pull/38551) the http-client throws error when the content-type were not occurate.

Which is the case for localstack **AND** for few endpoint in AWS. We once maintained a custom implementation of `toArray`, but when we removed this patch we forgot to bump the composer.json

**Review by Nyholm — APPROVED** [review]
Thank you!

**Comment by Nyholm** [comment]
Hm.. Tests seams to be failing: https://github.com/async-aws/core/runs/1791266599?check_suite_focus=true

**Comment by jderusse** [comment]
> Hm.. Tests seams to be failing: https://github.com/async-aws/core/runs/1791266599?check_suite_focus=true

Hmm.. I added a makefile, but forgot to call it in github action

**Comment by jderusse** [comment]
Fixed by #944

**Pull request merged**